src/Microsoft.ServiceFabric.Actors/Runtime/ActorStateProviderHelper.cs

```diff
@@ -496,6 +496,99 @@ internal static IActorStateProvider CreateDefaultStateProvider(ActorTypeInformat
             return Task.FromResult(actorQueryResult);
         }
 
+        internal Task<PagedResult<ActorId>> GetStoredActorIdsForKvsAsync(
+            int itemsCount,
+            ContinuationToken continuationToken,
+            KeyValueStoreReplica replica,
+            CancellationToken cancellationToken)
+        {

```

Comment:
Add a test case in windows fabric repo and paste the result of the local run here.

Reply:
Added

Reply:
PR for adding WinFab test case: https://msazure.visualstudio.com/One/_git/WindowsFabric/pullrequest/5531771